app/models/payment.rb

```diff
@@ -0,0 +1,23 @@
+class Payment < ApplicationRecord
+  ### CONSTANTS
+
+  STATUSES = {
+    paid: 0,
```

Comment:
DB default value is paid..this is not right

Reply:
Corrected